GameData/ROCapsules/PartConfigs/Apollo/ApolloDrogueParachute.cfg

```diff
@@ -7,9 +7,9 @@ PART
 	RSSROConfig = true
 	RP0conf = true
```

Comment:
Do not set RP-0 configuration state

Reply:
The ONLY thing that changed in this file is whitespace.  Please don't do that.  If you're making changes and want to fix whitespace around the change, great.  If it's enough to bother the entire file, great.  But just the tabs vs spaces of an entire file and nothing else is a pain to QC for minimal value.